Content.Client/ContextMenu/UI/EntityMenuElement.cs

```diff
@@ -31,33 +24,30 @@ public sealed partial class EntityMenuElement : ContextMenuElement
         /// <summary>
         ///     How many entities are accessible through this element's sub-menus.
         /// </summary>
-        /// <remarks>
-        ///     This is used for <see cref="CountLabel"/>
-        /// </remarks>
-        public int Count;
-
-        public readonly Label CountLabel;
-        public readonly SpriteView EntityIcon = new() { OverrideDirection = Direction.South};
+        private int _count;
+        public int Count {
+            get => _count;
+            private set {
+                _count = value;
+                if (_count > 1)
+                {
+                    IconLabel.Text = _count.ToString();
+                    IconLabel.Visible = true;
+                }
+            }
+        }
```

Comment:
Only ever set in UpdateCount, does not need encapsulated.

Reply:
Count is also set in the constructor and `Dispose` method. If I remove the setter I will have to change the default visibility of the `IconLabel`. Is this an acceptable solution for you ?

Reply:
I removed the custom setter. What do you think ?